cli/test/lib/tasks/install_spec.js

```diff
@@ -22,9 +22,18 @@ const packageVersion = '1.2.3'
 const downloadDestination = path.join(os.tmpdir(), 'cypress.zip')
 const installDir = '/cache/Cypress/1.2.3'
 
-describe('/lib/tasks/install', function () {
+const isCircle = Boolean(process.env.CIRCLECI)
+
+describe.only('/lib/tasks/install', function () {
```

Comment:
You left an `.only` in here

Reply:
updated `stop-only` to check multiple folders and now it will catch `.only` left in `packages` and in `cli` folders in the monorepo.